torch/csrc/Generator.cpp

```diff
@@ -68,6 +69,9 @@ static PyObject* THPGenerator_pynew(
     self->cdata = make_generator<MPSGeneratorImpl>();
   }
 #endif
+  else if (device.type() == at::kPrivateUse1) {
+    self->cdata = _register_genertor::GetGeneratorForPrivateuse1(device.index());
```

Comment:
small nit: I think it's fine if this just live directly in the `at` namespace, `at::GetGeneratorForPrivateUse1`.

Reply:
OK. thanks for your comments.

Reply:
@bdhirsh  Hi，I have modified my code based on your comments, could you review again?  I will be very very  
excited to merge it to pytorch, haha.